packages/gatsby-plugin-google-tag-manager/package.json

```diff
@@ -0,0 +1,31 @@
+{
+  "name": "@vtex/gatsby-plugin-google-tag-manager",
```

Comment:
isn't it better to call it of gatsby-plugin-pixel-tag-manager ?

Reply:
Humm, I've didn't find any references to the google-tag-manager being called a script.

With facebook they seem to use that name explicitly https://developers.facebook.com/docs/facebook-pixel/implementation

I was thinking about leveraging this opportunity to rethink how we name things. Since we don't have pixel builder/server anymore.

What are your thoughts @tlgimenes @klzns ?

Reply:
I agree, pixel was a term that we used but is not really that common. I would keep the name as is